### ISSUE TITLE
Сбой сеанса

### ISSUE THREAD

**Issue by FoxMul** [googlecolab/colabtools#2205]
Пытаюсь запустить следующий код:
----------------------------
### Load necessary libraries ###
import glob
import os
import librosa
import librosa.display as lbd
import numpy as np
import matplotlib.pyplot as plt
from matplotlib.pyplot import specgram
from sklearn.model_selection import KFold

import tensorflow as tf
from tensorflow import keras
plt.style.use('ggplot')

### Define helper functions ###
def load_sound_files(file_paths):
    raw_sounds = []
    for fp in file_paths:
        X,sr = librosa.load(fp)
        raw_sounds.append(X)
    return raw_sounds

def plot_waves(sound_names,raw_sounds):
    i = 1
    fig = plt.figure(figsize=(25,60), dpi=900)
    for n,f in zip(sound_names,raw_sounds):
        plt.subplot(10,1,i)
        librosa.display.waveplot(np.array(f),sr=22050)
        plt.title(n.title())
        i += 1
    plt.suptitle('Figure 1: Waveplot',x=0.5, y=0.915,fontsize=18)
    plt.show()

def plot_specgram(sound_names,raw_sounds):
    i = 1
    fig = plt.figure(figsize=(25,60), dpi = 900)
    for n,f in zip(sound_names,raw_sounds):
        plt.subplot(10,1,i)
        specgram(np.array(f), Fs=22050)
        plt.title(n.title())
        i += 1
    plt.suptitle('Figure 2: Spectrogram',x=0.5, y=0.915,fontsize=18)
    plt.show()

def plot_log_power_specgram(sound_names,raw_sounds):
    i = 1
    fig = plt.figure(figsize=(25,60), dpi = 900)
    for n,f in zip(sound_names,raw_sounds):
        plt.subplot(10,1,i)
        D = librosa.logamplitude(np.abs(librosa.stft(f))**2, ref_power=np.max)
        librosa.display.specshow(D,x_axis='time' ,y_axis='log')
        plt.title(n.title())
        i += 1
    plt.suptitle('Figure 3: Log power spectrogram',x=0.5, y=0.915,fontsize=18)
    plt.show()

def extract_feature(file_name):
    X, sample_rate = librosa.load(file_name)
    stft = np.abs(librosa.stft(X))
    mfccs = np.mean(librosa.feature.mfcc(y=X, sr=sample_rate, n_mfcc=40).T,axis=0)
    chroma = np.mean(librosa.feature.chroma_stft(S=stft, sr=sample_rate).T,axis=0)
    mel = np.mean(librosa.feature.melspectrogram(X, sr=sample_rate).T,axis=0)
    contrast = np.mean(librosa.feature.spectral_contrast(S=stft,
    	sr=sample_rate).T,axis=0)
    tonnetz = np.mean(librosa.feature.tonnetz(y=librosa.effects.harmonic(X),
    	sr=sample_rate).T,axis=0)
    return mfccs,chroma,mel,contrast,tonnetz

def parse_audio_files(parent_dir,sub_dir,file_ext='*.wav'):
    features, labels = np.empty((0,193)), np.empty(0) # 193 => total features
    for fn in glob.glob(os.path.join(parent_dir, sub_dir, file_ext)):
        mfccs, chroma, mel, contrast,tonnetz = extract_feature(fn)
        ext_features = np.hstack([mfccs,chroma,mel,contrast,tonnetz])
        features = np.vstack([features,ext_features])
        labels = np.append(labels, int(fn.split('/')[2].split('-')[1]))
    return np.array(features, dtype=np.float32), np.array(labels, dtype = np.int8)
sound_file_path = ["drive/MyDrive/Urban dataset/fold1/7061-6-0-0.wav","drive/MyDrive/Urban dataset/fold2/14387-9-0-7.wav",
                    "drive/MyDrive/Urban dataset/fold3/6988-5-0-3.wav","drive/MyDrive/Urban dataset/fold4/7064-6-3-0.wav",
                    "drive/MyDrive/Urban dataset/fold5/13577-3-0-0.wav","drive/MyDrive/Urban dataset/fold6/4912-3-1-0.wav",
                    "drive/MyDrive/Urban dataset/fold7/6902-2-0-4.wav",
                    "drive/MyDrive/Urban dataset/fold8/7390-9-0-0.wav","drive/MyDrive/Urban dataset/fold9/12812-5-0-0.wav",
                    "drive/MyDrive/Urban dataset/fold10/7965-3-3-0.wav"]
sound_names = ["air conditioner","car horn","children playing",
               "dog bark","drilling","engine idling",
               "gun shot","jackhammer","siren","street music"]
raw_sounds = load_sound_files(sound_file_path)
plot_waves(sound_names, raw_sounds)
plot_specgram(sound_names,raw_sounds)
---------------------------
Все работает нормально, кроме последних двух строчек. Если запускаю последнюю или предпоследнюю строку, то получаю сбой сеанса. Весь остальной код работает нормально.
Получаю сбой сеанса и в журнале вижу ошибку:
WARNING:root:kernel a04e1c34-95cc-44ba-89d5-b101711615ed restarted
Что это значит, и как это исправить? Гуглила, искала на гите, результатов не дало.
